mappings/net/minecraft/util/math/BlockBox.mapping

```diff
@@ -1,10 +1,18 @@
 CLASS net/minecraft/class_3341 net/minecraft/util/math/BlockBox
-	FIELD field_14376 maxZ I
-	FIELD field_14377 maxY I
-	FIELD field_14378 maxX I
-	FIELD field_14379 minZ I
-	FIELD field_14380 minY I
-	FIELD field_14381 minX I
+	COMMENT A mutable box with integer coordinates. The box is axis-aligned and the
+	COMMENT coordinates are inclusive.
+	COMMENT
+	COMMENT <p>This box, though mutable, has proper {@code hashCode} and {@code
+	COMMENT equals} implementations and can be used as map keys if user can ensure
+	COMMENT they are not modified.
+	COMMENT
+	COMMENT @see Box
+	FIELD field_14376 maxY I
+	FIELD field_14377 maxX I
+	FIELD field_14378 minZ I
+	FIELD field_14379 minY I
+	FIELD field_14380 minX I
+	FIELD field_14381 maxZ I
```

Comment:
These appear wrong. I recall these are a result of a bad match upon mojang's decision to no longer strip some code.